github/repos.go

```diff
@@ -655,10 +655,11 @@ func (s *RepositoriesService) GetRequiredStatusChecks(ctx context.Context, owner
 	p := new(RequiredStatusChecks)
 	resp, err := s.client.Do(ctx, req, p)
 	if err != nil {
-		return nil, resp, err
+		// if it's just a 404, don't return that as an error
+		_, err = parseBoolResponse(err)
 	}
```

Comment:
I think it'd be more readable to handle the error completely here, rather than continuing:

```Go
if err != nil {
	// if it's just a 404, don't return that as an error
	_, err = parseBoolResponse(err)
	return nil, resp, err
}
```